src/main/java/core/basesyntax/strategy/handler/OrderHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class OrderHandler implements OperationHandler {
```

Comment:
I don't see order operation in this list
![Screenshot 2022-05-23 at 11 36 44](https://user-images.githubusercontent.com/60258373/169778893-59b1a608-dfe5-4361-b2e4-7381d8bd7e0e.png)

Reply:
fixed